api/client/network.go

```diff
@@ -0,0 +1,9 @@
+package client
+
+import nwclient "github.com/docker/libnetwork/client"
+
+func (cli *DockerCli) CmdNetwork(args ...string) error {
+	nCli := nwclient.NewNetworkCli(cli.out, cli.err, nwclient.CallFunc(cli.call))
+	args = append([]string{"network"}, args...)
+	return nCli.Cmd("docker", args...)
```

Comment:
NetworkCli takes in the "chain" parameter for better --help reasons.
Since `libnetwork` is an ingredient that provides networking service to docker, but also it has its own independent tool named `dnet` (which does everything networking). The same CLI chain under `network` is reused for both the cases. Hence passing the actual chain the network UI belongs to helps with consistent and better --help output.